Automattic/wp-calypso
# Marketplace search: update translated fields

#### Proposed Changes

* Update the fields to accommodate the changes made in D83978-code and D84165-code
* Add language support to search results

#### Testing Instructions



* Patch D84165-code and sandbox public-api
* Switch your Calypso language on `/me/account`
* Enable the `marketplace-jetpack-plugin-search` flag
* Go to `/plugins/SITE` and search for a plugin
* Verify that the results are shown in the selected language

<img width="969" alt="CleanShot 2022-07-13 at 15 33 38@2x" src="https://user-images.githubusercontent.com/11555574/178746218-dcf62843-06f9-4288-b8c2-165523fb591c.png">



#### Pre-merge Checklist

Complete applicable items on this checklist **before** merging into trunk. Inapplicable items can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed.


Related to #64571
